### DISCUSSION TITLE
"Entity could not be found in cache for entity_id ....?

### DISCUSSION THREAD

**Discussion by misko2k** [zachowj/node-red-contrib-home-assistant-websocket#352] [Q&A]
Hi,

I have some strange problem , i have Chrome-Cast (last generation) connected on my TV , and i have 2 Google-Mini devices and 1 Soundbar , all devices i can find them under entity , but when i go under NodeRed, this (mediaplayer.telefunken) is not available in dropdown menu when i am searching for word "mediaplayer" ... 

I have few times restarted Hassio and even server ,but i still miss mediaplayer.telefonken

Any idea how to fix this ? 

![mediaplayer](https://user-images.githubusercontent.com/21138890/107644221-7254cc00-6c77-11eb-9b5c-52e814788f86.jpg)

**Comment by zachowj** [comment]
<https://zachowj.github.io/node-red-contrib-home-assistant-websocket/FAQ.html#entity-could-not-be-found-in-cache-for-entity-id>
